cpu/saml21/cpu.c

```diff
@@ -23,8 +23,42 @@
 
 static void _gclk_setup(int gclk, uint32_t reg)
 {
-    while (GCLK->SYNCBUSY.reg & GCLK_SYNCBUSY_GENCTRL(gclk)) {}
     GCLK->GENCTRL[gclk].reg = reg;
+    while (GCLK->SYNCBUSY.reg & GCLK_SYNCBUSY_GENCTRL(gclk)) {}
+}
+
+static void _osc32k_setup(void)
+{
+#if INTERNAL_OSC32_SOURCE
```

Comment:
please add #include "periph_conf." at the beginning of the file

Reply:
Ok.